src/components/Patient/CreatePatientForm.tsx

```diff
@@ -0,0 +1,226 @@
+import {
+    Button,
+    Checkbox,
+    DatePicker,
+    Form,
+    Input,
+    Select,
+    Upload,
+    UploadFile,
+    UploadProps,
+} from "antd";
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import { setPatientDetails } from "../../store/slice/patientInformationSlice";
+import "./createPatientForm.css";
+
+const { Item } = Form;
+const { Option } = Select;
+
+export const CreatePatientForm = () => {
+    const dispatch = useDispatch();
+    const [form] = Form.useForm();
+    const [fileList, setFileList] = useState<UploadFile[]>([]);
+
+    const [aggrementChecked, setAggrementChecked] = useState(false);
+
+    const handleForm = (values: any) => {
+        const formatedData = {
+            firstName: values.firstName,
+            lastName: values.lastName,
+            dob: values.dob.format("YYYY-MM-DD").toString(),
+            gender: values.gender,
+            ethnicity: values.ethnicity,
+            address: {
+                state: values.state,
+                street: values.street,
+                city: values.city,
+            },
+            payment: {
+                insuranceId: values.insuranceId,
+                memberId: values.memberId,
+                insuranceProvider: values.insuranceProvider,
+            },
+            document: values.document
+        };
+        dispatch(setPatientDetails(formatedData));
+
+        form.resetFields();
+    };
+
+    const uploadProps: UploadProps = {
+        onRemove: (file) => {
+            const index = fileList.indexOf(file);
+            const newFileList = fileList.slice();
+            newFileList.splice(index, 1);
+            setFileList(newFileList);
+        },
+        beforeUpload: (file) => {
+            setFileList([...fileList, file]);
+
+            return false;
+        },
+        fileList,
+    };
+
+    const handleAggrement = () => {
+        setAggrementChecked((prev: boolean) => !prev);
+    };
+
+    return (
+        <Form
+            labelAlign="left"
+            labelCol={{ span: 8 }}
+            onFinish={handleForm}
+            className="margin-top"
+        >
+            <Item
+                label="First name"
+                name="firstName"
+                rules={[{ required: true, message: "Please fill this field!" }]}
```

Comment:
if the message is similar through out the app, we can have it maintained in config file 
/src/config/appConfig.js 

FORM_DEFAULT_REQUIRED_MESSAGE = "Please fill this field!"